extension/src/utils/cpUtils.ts

```diff
@@ -3,15 +3,11 @@
 
 import * as cp from 'child_process';
 import * as kill from 'tree-kill';
-import { testOutputChannel } from '../testOutputChannel';
 
 export function killProcess(process: cp.ChildProcess): Promise<void> {
     return new Promise<void>((resolve: () => void): void => {
-        if (process) {
-            kill(process.pid, 'SIGTERM', (error: Error | undefined) => {
-                if (error) {
-                    testOutputChannel.error('Failed to kill the process', error);
-                }
+        if (process && !process.killed) {
+            kill(process.pid, 'SIGTERM', () => {
```

Comment:
If not output, at least log the issue for program diagnostic. It seems test runner need logger?

Reply:
@yaohaizh How about adding logger in the next release (0.12.0)